package.json

```diff
@@ -65,7 +65,7 @@
   },
   "peerDependencies": {
     "react": "^15.0.0-0 || ^16.0.0-0",
-    "react-dom": "^15.0.0-0 || ^16.0.0-0"
+    "react-dom": "^15.0.0-0 || ^16.0.0-0 || ^17.0.0-0"
```

Comment:
I checked [here](https://codesandbox.io/s/sleepy-glade-r3mf1?file=/src/App.js) with v17 and nothing seems to be breaking as of now. That being said, we will need to upgrade both react and react-dom if at a patch needs to be released.

Let me know if anything in particular is breaking due to this.

Reply:
Ok, I only know that npm failed to build my project due to this and gave me an error pointing to an "npm resolution report"

```
While resolving: web@0.1.0
Found: react@17.0.2
node_modules/react
  react@"^17.0.2" from the root project

Could not resolve dependency:
peer react@"^15.0.0-0 || ^16.0.0-0" from react-progressive-graceful-image@0.6.13
node_modules/react-progressive-graceful-image
  react-progressive-graceful-image@"*" from the root project

Fix the upstream dependency conflict, or retry
this command with --force, or --legacy-peer-deps
to accept an incorrect (and potentially broken) dependency resolution.
```

I decided to clone the repo, make the change and link it. 

I could missing something 🤷 as I am only just learning this whole frontend stack :)
(I have done almost purely backend for the last ~15 years)

Reply:
I couldn't find any issue while using this package with using react v17(latest). Still, I have made the changes to allow both `react` and `react-dom` ^v17 as peers. Both packages go together with the same version, usually. 

You can update to the new patch release [0.6.14](https://www.npmjs.com/package/react-progressive-graceful-image).
Thanks for pointing the issue and sending PR. Closing this now.
Hope it helps.